### ISSUE TITLE
Multiplus II GX: changes in utils.py have no effect 

### ISSUE THREAD

**Issue by SolarGuy724** [Louisvdw/dbus-serialbattery#494]
No matter how hard I try: I am not able to change charging current on my GX (Jiabaida BMS). 

Latest release dbus-serialbattery. 
dbus-serialbattery is installed and seems to work. All Venus menus as expected.
Limits are standard dbus-serialbattery 60/50 A due to venus.

System is working......

![funzt](https://user-images.githubusercontent.com/127744058/224689757-b8c26ce2-e60d-4da7-9943-44ed8b7dc497.jpg)

![screnn1](https://user-images.githubusercontent.com/127744058/224689897-3cd8fb80-0121-4fa7-b0bf-d69244307bac.jpg)

My problem:
Changes in util.py have no effect.  

![screnn2](https://user-images.githubusercontent.com/127744058/224689809-d1533363-ef3b-4c73-a31f-d342c6eb1470.jpg)

What I did:

- disable all 6 controls (i.e. in utils.py set all variables to False that end with _ENABLE). 
- changed in utils.py : CCL 100A and DCL 100A
- complete reinstall (clean-up and "fresh" install)
- reboot etc.

Can anyone please give me a hint what went wrong?

**Comment by SolarGuy724** [comment]
Issue solved! 

The problem was the starting-order in rc.local AFTER the ET340 Faker. Just moved it before ET340 Faker and the system works just fine :-).
Changes in utils.py now have the desired effects!

**Comment by skruehoetl** [comment]
Could you maybe give a few more details on you solution? I'm facing a similar-feeling issue with the jkBMS attached to the CerboGX of a friend. We keep trying to modify the CC max in the utils.py - and nothing happens.

**Comment by mr-manuel** [comment]
That makes no sense that the `/data/etc/dbus-serialbattery/utils.py` is overridden on each reboot.

Did you also check the charge current based on SoC?

**Comment by mr-manuel** [comment]
Ah, maybe it has to do something with the readonly mounts. Try to execute this before changing the `utils.py`. But I never heard before that this affects data in `/data`.

https://github.com/Louisvdw/dbus-serialbattery/blob/3685d15a5d1ba801867bbdb98f57fd805ea2c24a/etc/dbus-serialbattery/reinstalllocal.sh#L5-L6

**Comment by skruehoetl** [comment]
> That makes no sense that the `/data/etc/dbus-serialbattery/utils.py` is overridden on each reboot.
> 
> Did you also check the charge current based on SoC?

Yes. I just checked the current value in the HomeAssistant and at 90 % SoC the CCMax went down to 25 A (from 50 A - where I had set 100 A). I logged into the Cerbo and saw that `/data/etc/dbus-serialbattery/utils.py` is back to default again - so it contains settings for CCMax 50 A - so the 25 A match CC_CURRENT_LIMIT3.

**Comment by skruehoetl** [comment]
> Ah, maybe it has to do something with the readonly mounts. Try to execute this before changing the `utils.py`. But I never heard before that this affects data in `/data`.
> 
> https://github.com/Louisvdw/dbus-serialbattery/blob/3685d15a5d1ba801867bbdb98f57fd805ea2c24a/etc/dbus-serialbattery/reinstalllocal.sh#L5-L6

Okay... sounds intrigueing. I gave it a try and will report back as soon as my friend greenlights me rebooting his Cerbo...

**Comment by skruehoetl** [comment]
I have to apologize.

The problem is solved. I tried the remount-rw script before modifying the utils.py - and then I looked into the mounted volumes...
... and saw a SD-card in the list. Turns out, that we still had the SD-card from the original install in the slot and on each reboot, the driver happily kept overwriting itself.

Problem solved - and once more it only existed between keyboard and chair.

Sorry for bothering you and a big THANK YOU for your help!
